lib/flutter_secure_storage.dart

```diff
@@ -71,10 +71,10 @@ abstract class Options {
   }
 }
 
-class iOSOptions extends Options {
-  final String _groupId;
+class IOSOptions extends Options {
```

Comment:
This is a backwards incompatible change. Probably better to revert that and do it separately.